src/Tickets/Commerce/Gateways/PayPal/REST/Create_Order.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace TEC\Tickets\Commerce\Gateways\PayPal\REST;
+
+use TEC\Tickets\Commerce\Gateways\PayPal\Repositories\Order;
+
+/**
+ * Class Create_Order
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Commerce\Gateways\PayPal\REST
+ */
+class Create_Order {
+	/**
+	 * Create order.
+	 *
+	 * @since 5.1.6
+	 */
+	public function handle(  ) {
+		$postData = give_clean( $_POST );
+		$formId   = absint( tribe_get_request_var( 'give-form-id' ) );
+
+		$data = [
+			'formId'              => $formId,
+			'formTitle'           => give_payment_gateway_item_title( [ 'post_data' => $postData ], 127 ),
+			'paymentAmount'       => isset( $postData['give-amount'] ) ? (float) apply_filters( 'give_payment_total', give_maybe_sanitize_amount( $postData['give-amount'], [ 'currency' => give_get_currency( $formId ) ] ) ) : '0.00',
+			'payer'               => [
+				'firstName' => $postData['give_first'],
+				'lastName'  => $postData['give_last'],
+				'email'     => $postData['give_email'],
```

Comment:
Do these variables/indexes need to change?

Reply:
Well...I know they need to change. Should they change right now?

Reply:
Added all the WIP todos for remove these as @nefeline does the REST request handing this week.